assets/base/i18n/en.json

```diff
@@ -1482,6 +1482,12 @@
   "member_list.noUsersAdd": "No users to add.",
   "members_popover.msg": "Message",
   "members_popover.title": "Members",
+  "mobile.channel_info.publicChannel": "Public Channel",
```

Comment:
Oh sorry you are right, we can leave them.. didn't notice that one was for channel_info and the other ones for channel_list